charts/pega/values-large.yaml

```diff
@@ -233,6 +233,8 @@ global:
       # Create a stream tier for queue processing.  This tier deploys
       # as a stateful set to ensure durability of queued data. It may
       # be optionally exposed to the load balancer.
+      # Note: Stream tier is deprecated, please enable External Kafka configuration under External Services.
+      # When external Kafka is enabled, we should remove the entire stream tier.
       nodeType: "Stream"
```

Comment:
Can the stream tier creation is prevented if stram.enabled is set. See [here](https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/templates/_pega-deployment.tpl#L129).
Also it appears `stream.url` is being used in Pega Cloud deployments. Possibility of collisions. Do you foresee any issues with it?

Reply:
We can prevent it but it will break backward compatibility when we enable external kafka by default, when the external Kafka is enabled, the stream tier also connects to external Kafka. 
External kafka does not use stream.url so should be fine.

Reply:
Is this fine having both stream tier and external kakfa enabled?

Reply:
it is not recommended but if someone keeps both then external Kafka takes precedence and stream pods also connects to external Kafka